core/buffer.py

```diff
@@ -260,6 +260,9 @@ def update_with_data(self, update_data):
     def execute_function(self, function_name):
         getattr(self, function_name)()
 
+    def execute_function_with_arg(self, function_name, arg):
+        getattr(self, function_name)(arg)
+
```

Comment:
Instead of creating an additional function to execute with args, maybe we can reuse `execute_function` like this?
```
>>> class Class:
...     def foo(self, fname, *args):
...             getattr(self, fname)(*args)
```

Reply:
I think then we have to call all `execute_function` always with empty string as argument if it doesn't take any, but I haven't tested it. I will later look into it.

Reply:
> I think then we have to call all execute_function always with empty string as argument if it doesn't take any

Yes, `eaf-call` has to call `execute_function` with empty string.

Reply:
> I think then we have to call all execute_function always with empty string as argument if it doesn't take any

Yes, `eaf-call` has to call `execute_function` with empty string.